### PR TITLE
Made correction In Readme.md

### PR DESCRIPTION
**Pull request by AbhiiVops**
Corrected all the grammatical mistakes and punctuation in the readme.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,17 +67,17 @@ You can access the data by using this link:
...
 - Contact us page
 
 ## Expected JSON Data Format
 
-```json
+```JSON
 {
   "name": "Hydrogen",
-  "appearance": "colorless gas",
+  "Appearance": "colorless gas",
```
Comment (HarshKumarraghav) [inline README.md:80]: This is the data coming from the API so this remains the same. Correct this part then I will merge your PR.

File: README.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ You can access the data by using this link:
   "electron_affinity": 72.769,
   "electronegativity_pauling": 2.2,
   "ionization_energies": [1312],
-  "cpk-hex": "ffffff"
+  "CPC-hex": "ffffff"
```
Comment (HarshKumarraghav) [inline README.md:105]: The same goes for this.

File: README.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ You can access the data by using this link:
   "molar_heat": 28.836,
   "named_by": "Antoine Lavoisier",
   "number": 1,
-  "period": 1,
+  "Period": 1,
```
Comment (HarshKumarraghav) [inline README.md:91]: The same goes for this.

**Review by AbhiiVops — COMMENTED** [review]
Made relavant changes in the readme.md as suggested.

File: README.md (hunk 1 above)
Comment (AbhiiVops) [reply README.md]: I made relevant changes to the readme.md file. please see

File: README.md (hunk 2 above)
Comment (AbhiiVops) [reply README.md]: did this change too!

File: README.md (hunk 3 above)
Comment (AbhiiVops) [reply README.md]: and this one too

**Pull request merged**